pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -3346,6 +3346,17 @@ final: prev:
     meta.homepage = "https://github.com/junegunn/gv.vim/";
   };
 
+  hare-vim = buildVimPluginFrom2Nix {
+    pname = "hare.vim";
+    version = "2022-07-02";
+    src = fetchFromSourceHut {
+      url = "https://git.sr.ht/~sircmpwn/hare.vim";
+      rev = "5c758cdbbabd6e4ba92bced9428cd1fa4212f003";
+      sha256 = "0l1q9x1n30h32zfnd2krg3frywydfmb8ic05619pylamnyh4w918";
+    };
```

Comment:
```suggestion
    src = fetchFromSourcehut {
      owner = "~sircmpwn";
      repo = "hare.vim";
      rev = "5c758cdbbabd6e4ba92bced9428cd1fa4212f003";
      sha256 = "0l1q9x1n30h32zfnd2krg3frywydfmb8ic05619pylamnyh4w918";
    };
```

Reply:
Right! Not sure what I did testing this that it worked :sweat: 

Updated correctly now.